### PR TITLE
amenities detail columns added Space model

### PR DESCRIPTION
**Pull request by frank657**

### CODE REVIEW

**Comment by juliakimmig** [comment]
Looks great Frank 👍🏻

File: db/schema.rb [hunk 1]
```diff
@@ -29,12 +29,16 @@
     t.string "name"
     t.string "district"
     t.string "address_details"
-    t.string "amenities"
     t.string "picture"
     t.integer "price"
```
Comment (grantye) [inline db/schema.rb]: Good

File: db/schema.rb [hunk 2]
```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2019_04_01_091200) do
```
Comment (juliakimmig) [inline db/schema.rb:13]: This line is red

**Review by grantye — COMMENTED** [review]
Comment

**Review by juliakimmig — COMMENTED** [review]
comment

**Pull request merged**